content/tbd/Dark Matter Labs.md

```diff
@@ -0,0 +1,55 @@
+---
+created: 2024-02-28
+contributors:
+  - Elisa Paka
+title: Dark Matter Labs
+url: https://darkmatterlabs.org/
+locations:
+  - Amsterdam, Netherlands
+  - London, UK
+  - Montréal, Canada
+  - Seoul, South Korea
+  - Malmö, Sweden
+start: 
+end: 
+size: 51-100
+image: ../assets/dark-matter-labs.png
+sectors:
+  - Economics & Business
+  - Education & Learning
+  - Governance & Community
+activities:
+  - Design, Development & Innovation
+  - Research
+cohere: true
+metadata_version: "1"
+---
+![[../assets/dark-matter-labs.png]] 
+
+> Dark Matter Labs (Dm) is working on what is needed to manifest transformations to our food, housing, land, material and nature systems towards a future of mutual thriving.  
+> 
+> In doing this, we analyse the shifts it requires in the underlying ‘dark matter’ - monetary, economic, governance, regulatory and policy systems - to make this possible, and we work with partners to demonstrate these alternatives in neighbourhoods, cities and bioregions. We share these insights openly for mutual learning.
+
+## Key People
+
+## Theory of Change
+
+> We’re establishing toolkits and blueprints, pilots, and case studies, supporting communities and institutions with applications, digital products and civic technologies that challenge established thought and demonstrate that an alternative is possible.  
```

Comment:
this stuff i'd probably put above in the about/description section, as opposed to theory of change section, as it seems to me more about their activities/what they're doing as opposed to their theory or strategy or underlying principles of change.